.gitlab-ci.yml

```diff
@@ -167,7 +167,7 @@ after_script:
   dependencies:
     - not-a-real-job
   script:
-    - cd _install_ci
+    - cd _install_ci || exit 0 # maybe no artifacts from allow_failure dep on build:quick
```

Comment:
Umm, I am not a fan of this given that it will apply to all validate jobs; shouldn't we instead mark that particular job as allow fail then?

Why is gitlab trying to start the validate job with a failed dependency?

Reply:
>shouldn't we instead mark that particular job as allow fail then?

It's not allow_fail though, if build:quick succeeds then we expect this one to succeed too as it should only operate on fairly normal vo files
>Why is gitlab trying to start the validate job with a failed dependency?

Because that's how the semantics of allow_failure are 🤷

Reply:
I see, thanks; but still the scope of the patch seems to wide for me; we could have a problem with the general CI setup for validate and it would be undetected for days.

Can we restrict the fix [with a var] to the particular job involved?

In fact this illustrates the very high cost of carrying unreliable tests in the CI, we should just drop the tests responsible for this.

Reply:
In fact none of the base builds should be in allow failure; again we are adding hack to the CI to accommodate for broken code; IMO the broken is the thing that should be fixed.